Hazel/src/Platform/OpenGL/OpenGLVertexArray.cpp

```diff
@@ -54,11 +54,16 @@ namespace Hazel {
 		vertexBuffer->Bind();
 
 		uint32_t index = 0;
+		uint32_t count = 0;
+		for(auto a : m_VertexBuffers)
```

Comment:
Mostly coding style review here:
1. Might be best to store a referece
    `auto& a`
2. Can be const
    `cont auto& a`
3. Use decent names, for example
    `const auto& vertexBuffer : m_VertexBuffers`